docs/development.rst

```diff
@@ -27,16 +27,7 @@ For the same reason, we currently recommend running PyCharm from wsl rather than
     $ git clone https://github.com/comic/grand-challenge.org
     $ cd grand-challenge.org
 
-3. Add the following to your hosts file (``/etc/hosts`` on Linux, ``C:\Windows\System32\drivers\etc\hosts`` on Windows):
-
-.. code-block:: console
-
-    127.0.0.1 gc.localhost
```

Comment:
I tried removing this and in my browser `gc.localhost` was still accessible, but `ping gc.localhost` in my terminal fails and trying to access the api with python requests also fails:
```
>>> import requests
>>> r = requests.get('https://gc.localhost/api/v1/profiles/users/', verify=False)
Traceback (most recent call last):
  File "c:\python37\lib\site-packages\urllib3\connection.py", line 171, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw)
  File "c:\python37\lib\site-packages\urllib3\util\connection.py", line 56, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "C:\Python37\lib\socket.py", line 748, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\python37\lib\site-packages\urllib3\connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "c:\python37\lib\site-packages\urllib3\connectionpool.py", line 343, in _make_request
    self._validate_conn(conn)
  File "c:\python37\lib\site-packages\urllib3\connectionpool.py", line 849, in _validate_conn
    conn.connect()
  File "c:\python37\lib\site-packages\urllib3\connection.py", line 314, in connect
    conn = self._new_conn()
  File "c:\python37\lib\site-packages\urllib3\connection.py", line 180, in _new_conn
    self, "Failed to establish a new connection: %s" % e)
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x00000211967942E8>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\python37\lib\site-packages\requests\adapters.py", line 445, in send
    timeout=timeout
  File "c:\python37\lib\site-packages\urllib3\connectionpool.py", line 638, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "c:\python37\lib\site-packages\urllib3\util\retry.py", line 398, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='gc.localhost', port=443): Max retries exceeded with url: /api/v1/profiles/users/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x00000211967942E8>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "c:\python37\lib\site-packages\requests\api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "c:\python37\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "c:\python37\lib\site-packages\requests\sessions.py", line 512, in request
    resp = self.send(prep, **send_kwargs)
  File "c:\python37\lib\site-packages\requests\sessions.py", line 622, in send
    r = adapter.send(request, **kwargs)
  File "c:\python37\lib\site-packages\requests\adapters.py", line 513, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='gc.localhost', port=443): Max retries exceeded with url: /api/v1/profiles/users/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x00000211967942E8>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))
```

Reply:
Yes, I guess that's correct as mapping localhost subdomains is a browser feature, not an OS feature. It was Chrome only but is now added in Firefox 84 (https://bugzilla.mozilla.org/show_bug.cgi?id=1220810) and it looks like Safari does not support it (https://bugs.webkit.org/show_bug.cgi?id=160504) but none of us develop on that. Do we often need it to work from cURL? I can add it back but to a different section in the documentation if so, nice that we get to remove 1 step for most use cases.

Reply:
Indeed probably better to add it in a different section. I use it for running my custom script after setup that uploads retina archives/patients/images etc to the retina importer endpoints. But other then that it is probably not used often.